src/services/accounts/AccountsBalanceInfoService.ts

```diff
@@ -25,9 +25,11 @@ export class AccountsBalanceInfoService extends AbstractService {
 		hash: BlockHash,
 		historicApi: ApiDecoration<'promise'>,
 		address: string,
-		token: string
+		token: string,
+		balanceToHuman: boolean
 	): Promise<IAccountBalanceInfo> {
 		const { api } = this;
+		const decimal = api.registry.chainDecimals[0];
```

Comment:
Do we know that this value will always be there for a substrate based chain, or is it safe to say we should check and throw an error if it doesnt exist?

Reply:
if a substrate base chain is using a SS58 address it will always exist AFAIU the list of addresses can be found https://github.com/paritytech/ss58-registry/blob/main/ss58-registry.json

however it's possible for a substrate based chain to use some other format.

Reply:
> https://github.com/paritytech/ss58-registry/blob/main/ss58-registry.json

This is extremely helpful! Thanks so much. Yea based on seeing some of them having no decimal value within the array, I am going to throw an error for those who use the query param against a chain that doesnt have the value available. One of them in the list for example: subsocial.

Reply:
(I wonder why the symbol and decimals are in an array; it sortof implies that there could be more than 1, too, in some cases?)

Reply:
Yea you are correct. After doing some research though, the order of the values in the array correspond to the values in the tokens array: So... If the available tokens in the chain are `['MOVR', 'GMBR']` -> then the chainsDecimals array would look like `[18, 18]`. I haven't seen a single case where the decimals change, BUT doesnt mean it cant. I should probably check for this.